src/index.js

```diff
@@ -6,11 +6,19 @@ class Handler {
 
   handle(message){
     if(message instanceof this.messageType){
-      this.callback.call(null, message);
+      executeHandler(() => this.callback.call(null, message));
     }
   }
 }
 
+function executeHandler(handler) {
+  try {
+    handler();
+  } catch(e) {
+    console.error(e);
```

Comment:
It'd be nice to support the aurelia/logging... should we add the dependency? Or allow an optional logger to be specified in the ctor?